.env.sample

```diff
@@ -1,10 +1,20 @@
-##### Required value
+##### Required values
 # Django Secret key. Run the command below to generate and insert this.
 # You should set this to production but the start script generates one each time if not set
 
 # python -c 'from django.core.management.utils import get_random_secret_key; print(get_random_secret_key())'
 # DJANGO_SECRET_KEY=<some secret key, randomized by default>
 
+# Canvas OAuth Config
```

Comment:
During the testing with localhost, we find it is necessary to set "CSRF_COOKIE_SECURE=True" in the .env file. 

Need to add CSRF_COOKIE_SECURE entry into the .env.sample file, since the default value is False if missing.

Reply:
Created this as a new issue. I'll check out the properties needed.